core/src/init/init.ts

```diff
@@ -13,3 +13,4 @@ import "./pane/pane";
 import "./popover/popover";
 import "./select/select";
 import "./tab/tab";
+import "./category/category";
```

Comment:
The "category" module is used by other components (e.g. Tag) so it's better to put it at the very top of these imports. I suggest to move it to above the import of "icon"